source/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ add_subdirectory(module_base)
 add_subdirectory(module_cell)
 add_subdirectory(module_symmetry)
 add_subdirectory(module_neighbor)
+add_subdirectory(module_psi)
+add_subdirectory(module_elecstate)
+add_subdirectory(module_hamilt)
+add_subdirectory(module_hsolver)
```

Comment:
Personally, I suggest naming refactored module without the module- prefix to distinguish them from former code. What do you think?

Reply:
My prefer is to change these directory name in future, but not this time.